packages/ui/components/ui/block-sync-status/large.tsx

```diff
@@ -47,6 +49,38 @@ const BlockSyncErrorState = ({ error }: { error: unknown }) => {
   );
 };
 
+// Not having a fullSyncHeight indicates we are doing a fresh sync.
+// The genesis block is particularly large, so we have special text to indicate that.
+const AwaitingState = ({ genesisSyncing }: { genesisSyncing: boolean }) => {
+  return (
+    <motion.div
+      className='flex w-full flex-col gap-1'
+      initial={{ opacity: 0 }}
+      animate={{ opacity: 1, transition: { duration: 0.3, ease: 'easeOut', delay: 0.75 } }}
```

Comment:
Set a short delay. If not, flickers the screen.

Reply:
Not sure I understand what flickers, but setting a hard-coded time delay to avoid flicker feels a little hacky to me, since whether it works depends on the speed of the user's computer. Is there not a deterministic way to prevent flicker?

Reply:
Here's without it. Essentially, we need a sort of UI debouncer to prevent it. I found a slight animation delay is sufficient, but perhaps there is a better way 🤔 

https://github.com/penumbra-zone/web/assets/16624263/5af95a28-a00d-42ef-a295-4a7822f5768d